authui/src/date.ts

```diff
@@ -130,15 +131,18 @@ export function formatDateRelative() {
 // So the comprimise is to make the display format of such value matches that of <input type="date">.
 // The display format of <input type="date"> is in browser locale for Safari and Firefox.
 // For Chrome, the display format is somehow arbitrary :(
-export function formatInputDate() {
-  const hasAbs = intlDateTimeFormatIsSupported();
-  if (!hasAbs) {
-    return;
-  }
+export class FormatInputDateController extends Controller {
+  static targets = ["inputDate"];
 
-  const dateSpans = document.querySelectorAll("[data-input-date-value]");
-  for (let i = 0; i < dateSpans.length; i++) {
-    const dateSpan = dateSpans[i];
+  declare inputDateTarget: HTMLSpanElement;
+
+  connect() {
+    const hasAbs = intlDateTimeFormatIsSupported();
+    if (!hasAbs) {
+      return;
+    }
+
+    const dateSpan = this.inputDateTarget;
```

Comment:
This original content should be restored when disconnect. Otherwise, connecting twice will permanently lose the original content.